nodes/index.adoc

```diff
@@ -34,13 +34,13 @@ image::295_OpenShift_Nodes_Overview_1222.png[Overview of control plane and worke
 
 The read operations allow an administrator or a developer to get information about nodes in an {product-title} cluster.
 
-ifndef::openshift-rosa-hcp,openshift-rosa[]
+ifndef::openshift-enterprise,openshift-rosa-hcp,openshift-rosa[]
 * xref:../nodes/nodes/nodes-nodes-viewing.adoc#nodes-nodes-viewing-listing_nodes-nodes-viewing[List all the nodes in a cluster].
 * Get information about a node, such as memory and CPU usage, health, status, and age.
 * xref:../nodes/nodes/nodes-nodes-viewing.adoc#nodes-nodes-viewing-listing-pods_nodes-nodes-viewing[List pods running on a node].
-endif::openshift-rosa-hcp,openshift-rosa[]
+endif::openshift-enterprise,openshift-rosa-hcp,openshift-rosa[]
 
-ifndef::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
+ifndef::openshift-enterprise,openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
```

Comment:
I'm not sure I understand the conditionalization in this file. Some of it (like this instance) seems to exclude content from all OpenShift distros.

Reply:
These sections weren't working at all and I figured I'd push fixing it to another PR.

Reply:
:+1: I'm currently working on the Nodes content. Perhaps that will automagically fix it. (We can always dream...)